lib/compat/wordpress-6.2/block-editor-settings.php

```diff
@@ -1,31 +0,0 @@
-<?php
-/**
- * Adds settings to the block editor.
- *
- * @package gutenberg
- */
-
-/**
- * Adds styles and __experimentalFeatures to the block editor settings.
- *
- * @param array $settings Existing block editor settings.
- *
- * @return array New block editor settings.
- */
-function gutenberg_get_block_editor_settings_6_2( $settings ) {
```

Comment:
Documenting what I learn: this has been moved to `lib/block-editor-settings.php`, which includes this code (and much more).